src/accommodation/accommodation.controller.ts

```diff
@@ -62,13 +64,13 @@ export class AccommodationController {
   @LangQuery()
   @Post('/')
   async createAccommodation(
-    @Body() body: CreateAccommodationAndAddressDto,
+    @Body() body: CreateAccommodationDto,
     @CurrentUser('id') userId: string
   ) {
     const createAccommodationAndAdress = {
```

Comment:
I don't think 'create' prefix is necessary, cuz it's not an action

Reply:
i got two DTOs, one fro creating, there most of fields are requered, and the second one is updating where all of the fields are optinal.
i do not know how to name them properly if i need to remove create and update prefixes

Reply:
gotcha. ok then